syz-ci/manager.go

```diff
@@ -513,6 +513,8 @@ func (mgr *Manager) createTestConfig(imageDir string, info *BuildInfo) (*mgrconf
 	if err := mgrconfig.Complete(mgrcfg); err != nil {
 		return nil, fmt.Errorf("bad manager config: %v", err)
 	}
+	// We don't need asset storage for test runs.
+	mgrcfg.AssetStorage = nil
```

Comment:
The error is produced by the Complete function. Isn't it too late to reset it here?
But also we get AssetStorage/Dashboard settings from mgr.managercfg, right? Why didn't we get the same error for mgr.managercfg?

Reply:
> The error is produced by the Complete function. Isn't it too late to reset it here?

Yes, that's a problem. Thanks!

> Why didn't we get the same error for mgr.managercfg?

Because we first loaded the config from the raw array and then adjusted it here
https://github.com/google/syzkaller/blob/77d3f68986094b5a8d09dcf4f3b84008e9d1962f/syz-ci/syz-ci.go#L406

I think it's worth it to do an extra config validation at the bottom of that method to catch such problems earlier.

Reply:
Or, maybe, I can just move asset storage config setting to `writeConfig`

Reply:
Good idea. It's reasonable to set AssetConfig in the same function where we set Dashboard in the config.

Reply:
PTAL